README.md

```diff
@@ -50,7 +50,15 @@ Regenerating critical CSS
 Running the tests
 -----------------
 
-`docker-compose run app vendor/bin/phpunit`
+To run all unit tests as in `ci`:
+
+`docker-compose -f docker-compose.yml -f docker-compose.ci.yml run ci .ci/phpunit`
+
+To run a single test:
+
+```
+APP_ENV=ci docker-compose run app vendor/bin/phpunit --filter AboutControllerTest
```

Comment:
This didn't work for me, but `APP_ENV=ci docker-compose  -f docker-compose.yml -f docker-compose.ci.yml run ci vendor/bin/phpunit --filter AboutControllerTest` did. 

The issue was missing phpunit in `vendor`, so my guess is that it's my vendor folder being empty and then getting bind mounted over the top of the image?

Reply:
I will investigate and improve. Thanks.